src/js/angular/core/services/connectors.service.js

```diff
@@ -13,44 +13,61 @@ ConnectorsService.$inject = ['ConnectorsRestService'];
 function ConnectorsService(ConnectorsRestService) {
 
     /**
-     * Fetches all connector types from the server.
+     * Fetches all connector types from the server for the repository with id <code>repositoryId</code> and location <code>repositoryLocation</code>.
+     * If the repository ID and repository location are not provided, the currently selected repository will be used.
+     *
+     * @param {string} repositoryId - (optional) The repository id.
+     * @param {string} repositoryLocation - (optional) The repository location.
      * @return {Promise<ConnectorTypesListModel>}
      */
-    const getConnectorTypes = () => {
-        return ConnectorsRestService.getConnectors()
+    const getConnectorTypes = (repositoryId, repositoryLocation) => {
+        return ConnectorsRestService.getConnectors(repositoryId, repositoryLocation)
             .then((response) => connectorTypesListMapper(response.data));
     };
 
     /**
-     * Fetches the connector prefix by its name.
+     * Fetches the connector prefix by its name for the repository with id <code>repositoryId</code> and location <code>repositoryLocation</code>.
+     * If the repository ID and repository location are not provided, the currently selected repository will be used.
+     *
      * @param {string} name
+     * @param {string} repositoryId - (optional) The repository id.
+     * @param {string} repositoryLocation - (optional) The repository location.
```

Comment:
What does it mean `(optional)` for these 2 parameters? Shouldn't optional parameters have default values?

Reply:
Done